waspc/data/Generator/templates/sdk/wasp/server/oauth/oneTimeCode.ts

```diff
@@ -0,0 +1,50 @@
+import { createJWT, validateJWT, TimeSpan } from '../../auth/jwt.js'
+
+export const tokenStore = createTokenStore();
```

Comment:
We need the one time tokens to allow the client to exchange one time codes for session IDs. If the server and client were on the same domain, it would be possible. This is not the case right now, so we need this kind of a singleton that keeps track of the tokens. This is not introduced in this PR btw.

Reply:
It's not a singleton in a sense that there can only exist only one instance of the object. It's true that we create a single instance that we _have to use_ in both places where it's needed. In the case of unit tests, we could make the token store a param of the which ever fn we wanted to unit test.